kmapper/visuals.py

```diff
@@ -210,13 +211,10 @@ def format_mapper_data(
     return json_dict
 
 
-def build_histogram(data, colorscale=None, nbins=10):
+def build_histogram(data, colorscale=colorscale_default, nbins=10):
```

Comment:
can leave like this, or go back to `None` or use `_return_default_colorscale`

Reply:
The `None` pattern I think will render better in the documentation.